cmd/brew-test-bot.rb

```diff
@@ -1464,10 +1464,10 @@ def test_ci_upload(tap)
 
     if git_tag
       if ARGV.include?("--dry-run")
-        puts "git push --force #{remote} master:master :refs/tags/#{git_tag}"
+        puts "git push --force #{remote} origin/master:master :refs/tags/#{git_tag}"
       else
-        safe_system "git", "push", "--force", remote,
-                                   "master:master", ":refs/tags/#{git_tag}"
+        safe_system "git", "push", "--force", remote, "origin/master:master",
+                                                      ":refs/tags/#{git_tag}"
```

Comment:
I have a mild preference for the original formatting, with the list of branches on the same line.

Reply:
Sure I'll change it. Where to you want the branch line to start? at `"--force"` or at `remote`?

Reply:
```
safe_system "git", "push", "--force", remote,
                           "origin/master:master", ":refs/tags/#{git_tag}"
```

or 

```
safe_system "git", "push", "--force", remote,
                                      "origin/master:master", ":refs/tags/#{git_tag}"
```

Reply:
and should I change it here as well?

https://github.com/Homebrew/homebrew-test-bot/blob/master/cmd/brew-test-bot.rb#L1561

Reply:
I thought that `brew style` should be opinionated about this choice. I believe it should line up with `"git"`. (My personal preference is just a normal two space indent, but `brew style` doesn't like that.)

Reply:
> and should I change it here as well?
> https://github.com/Homebrew/homebrew-test-bot/blob/master/cmd/brew-test-bot.rb#L1561

No line break if it fits nicely on one line (fewer than 100 characters).

Reply:
> No line break if it fits nicely on one line (fewer than 100 characters).

both lines are exactly the same except for one `:` in the first one.

And they are both shorter than 100 characters.

Soooo.... should I keep it as it is in master?

Reply:
Yeah, keep it as in master.